lib/nerves_hub_link/health_check/report.ex

```diff
@@ -1,7 +1,7 @@
 defmodule NervesHubLink.HealthCheck.Report do
   alias NervesHubLink.Message.DeviceStatus.Peripheral
 
-  @callback timestamp() :: DateTime.t()
+  @callback timestamp() :: String.t()
```

Comment:
This was intentional. They serialize well and this means we avoid sending bad timestamps.

Reply:
Dialyzer was exploding in CI due to this. You can switch it back, I just went for the quick fix.

Reply:
What's wrong with it according to dialyzer?